tests/group_functions/group_reduce_fp64.cpp

```diff
@@ -0,0 +1,223 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright (c) 2023 The Khronos Group Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+*******************************************************************************/
+
+#include "group_reduce.h"
+
+// FIXME: ComputeCpp does not implement reduce for unsigned long long int and
+//        long long int
+#ifdef SYCL_CTS_COMPILING_WITH_COMPUTECPP
+#ifdef SYCL_CTS_ENABLE_FULL_CONFORMANCE
+using ReduceTypes =
+    unnamed_type_pack<size_t, float, char, signed char, unsigned char,
+                      short int, unsigned short int, int, unsigned int,
+                      long int, unsigned long int>;
+#else
+using ReduceTypes = unnamed_type_pack<float, char, int>;
+#endif
+#else
+using ReduceTypes = Types;
+#endif
+
+using DoubleExtendedTypes = concatenation<ReduceTypes, double>::type;
+// 2-dim Cartesian product of type lists
+using prod2 =
```

Comment:
At the end it is unclear about what is the type of `prod2` here.
At least give a hint about the content.

Reply:
See a comment above.